plugin/storage/s3/s3_test.go

```diff
@@ -23,58 +31,56 @@ var (
 	}
 )
 
-func TestListObjects(t *testing.T) {
+func TestS3Operations(t *testing.T) {
 	t.Skip()
 	a := require.New(t)
 	ctx := context.Background()
-	client, err := NewClient(ctx, region, bucket, credentials)
+	credentialsFileName := writeTempCredentialsFile(t, credentials)
+	client, err := NewClient(ctx, region, bucket, credentialsFileName)
```

Comment:
A bit inconvenient to pass the filename since it requires us to write a temp file. Is this a design limitation of the underlying aws golang lib?

Reply:
Not really, will refactor.